### ISSUE TITLE
Reachy force sensor high values

### ISSUE THREAD

**Issue by louisborn** [alpakadev/robot-ludens#84]
Reading the force sensor of reachy is returning very high values (million)

Code used:

`        if self.reachy.force_sensors.r_force_gripper.force > constants.GRIP_FORCE_HOLDING:`

**Comment by louisborn** [comment]
![PXL_20230620_130242340.jpg](https://github.com/alpakadev/robot-ludens/assets/56700857/54da0a64-e96b-4bb3-87d9-b5e415b83900)

![PXL_20230620_130116797.jpg](https://github.com/alpakadev/robot-ludens/assets/56700857/6719aecc-cdd6-4893-99e2-19d81cba6649)

**Comment by alpakadev** [comment]
There is an [Answer on the Pollen Robotics Discord](https://discord.com/channels/519098054377340948/896070016049491988/1123256248704450661)  on this issue, provided the copied text below:

> It’s normal to currently having six to seven digits for the force sensor value on a Reachy 2023, the value being the bytes value directly casted to int.

For Reachy 2021, we applied a conversion from the bytes value to get a value more readable as shown in the documentation: 
https://docs.pollen-robotics.com/sdk/first-moves/arm/#from-the-force-sensor

The conversion was done on Reachy 2021 on small embedded boards use to communicate with the motors and other sensors on the robot. Because on the 2023 version we are using anymore these boards, we forgot to perform this conversion somewhere else. We will make a patch including the conversion for the value of the Reachy 2023 as well.

**However**, please note that because the readings for the force sensor are noisy, you cannot bet on the fact that the value reading will be reproducible. For example, if you grasp an object with the gripper, read the value and grasp the same object the next day, you have no guarantee that you will get the exact same value. The readings will vary from one sensor to another and even from one use of the robot to another. Maybe you can try a calibration by closing the gripper and reading the force sensor value but this probably will have to be done each time you use the robot.

TLDR; the force sensor readings relative values can be used, but not the absolute ones.
